server/testserver.go

```diff
@@ -32,6 +35,7 @@ import (
 )
 
 // StartTestServer starts a in-memory test server.
+// Adds a permissions config for "test-user" under prefix "test-user".
```

Comment:
I think anyone using Start manually will have a very good reason to do so. I've done it a couple of times, but only when I needed a custom context, or other weird settings before startup.